src/main/java/net/minecraftforge/common/util/EnumHelper.java

```diff
@@ -141,6 +143,18 @@ public static EnumRarity addRarity(String name, TextFormatting color, String dis
     {
         return addEnum(EnumRarity.class, name, color, displayName);
     }
+    /**
+     * 
+     * @param name - The name the new enum constant should have
```

Comment:
Don't say "the name the new enum constant should have," say "the name of the new {<span>@</span>code HorseArmorType}".